### ISSUE TITLE
Random fail for message deleting

### ISSUE THREAD

**Issue by poupotte** [cozy/cozy-emails#439]
I tried to delete a message with keyboard shortcut. I received this error :
```
"TypeError: account is undefined" "https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js" 11775 4 TypeError: account is undefined
Stack trace:
AccountStore</AccountStore.prototype._applyMailboxDiff@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:11775:5
MessageStore</onReceiveRawMessage@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:12460:16
MessageStore</MessageStore.prototype.__bindHandlers/<@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:12504:9
Store</_processBinding/this.dispatchToken</<@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:10119:18
Dispatcher</Dispatcher.prototype._invokeCallback@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:9936:5
Dispatcher</Dispatcher.prototype.dispatch@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:9906:23
AppDispatcher</AppDispatcher.prototype.handleViewAction@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:1128:5
_localDelete@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:1021:1
MessageActionCreator.delete@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:742:16
module.exports.messageDeleteCurrent@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:13232:1
root.mailkeys._binding.d.action@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44505:11
x@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44591:439
m.handleKey@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44597:1
y@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44592:144
 "AccountStore</AccountStore.prototype._applyMailboxDiff@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:11775:5
MessageStore</onReceiveRawMessage@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:12460:16
MessageStore</MessageStore.prototype.__bindHandlers/<@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:12504:9
Store</_processBinding/this.dispatchToken</<@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:10119:18
Dispatcher</Dispatcher.prototype._invokeCallback@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:9936:5
Dispatcher</Dispatcher.prototype.dispatch@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:9906:23
AppDispatcher</AppDispatcher.prototype.handleViewAction@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:1128:5
_localDelete@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:1021:1
MessageActionCreator.delete@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:742:16
module.exports.messageDeleteCurrent@https://bellot-zoe.cozycloud.cc/apps/emails/js/app.js:13232:1
root.mailkeys._binding.d.action@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44505:11
x@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44591:439
m.handleKey@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44597:1
y@https://bellot-zoe.cozycloud.cc/apps/emails/js/vendor.js:44592:144
```

I retried and it worked.

This behavior happens for two differents messages (in two differents mailboxes).

**Comment by clochix** [comment]
Probably fixed by b439414

**Comment by poupotte** [comment]
After several deleting, I didn't receive errors.
It seems fixed. Thanks @clochix .
